hr_employee_health/models/hr_employee.py

```diff
@@ -0,0 +1,26 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    health_condition_ids = fields.One2many(
+        string='Health Conditions',
+        comodel_name='hr.employee.health.condition',
+        inverse_name='employee_id',
+        groups='hr.group_hr_user',
+        help='Health condition of an employee.'
+    )
+    blood_type = fields.Many2one(
```

Comment:
```suggestion
    blood_type_id = fields.Many2one(
```

Reply:
You cant just change a field name like this. If you want to do it you have to add a migration script. I would keep the name as it was...

Reply:
Indeed, I would suggest doing this (Including Migration script).
One has to take the flag and correct all these. (As this doesn't respect the guidelines)

Reply:
Ok then! :wink:

Reply:
This indeed should be fixed.